Samples/iOS-Swift/iOS-SwiftUITests/TopViewControllerTests.swift

```diff
@@ -0,0 +1,89 @@
+import XCTest
+
+class TopViewControllerTests: XCTestCase {
```

Comment:
`m`: When you use `BaseUITest` added with https://github.com/getsentry/sentry-cocoa/pull/3373 you can remove the `setUp`, `tearDown` and `waitForExistenceOfMainScreen`.